.travis.yml

```diff
@@ -0,0 +1,24 @@
+job:
+  include:
+    - stage: Unit tests
+      name: Unit tests SDK Kotlin
+      language: kotlin
+      jdk: openjdk8
+      sudo: false
+      before_cache:
+        - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+        - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+      cache:
+        directories:
+          - "$HOME/.gradle/caches/"
+          - "$HOME/.gradle/wrapper/"
+      install:
+        - cd $TRAVIS_BUILD_DIR/
+      script:
+        - ./gradlew test
+
+stages:
+  - name: Unit Tests
+    if: type =~ /(cron|push|pull_request)/ AND branch =~ /^master|[0-9]+-(dev|stable)$/
+  - name: Builds
+    if: type =~ /(cron|push|pull_request)/ AND branch =~ /^master|[0-9]+-(dev|stable)$/
```

Comment:
I think you do not need this stage for now :slightly_smiling_face:

Reply:
I leave it here so we don't forget it later